org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -515,11 +519,36 @@ SsoApplicationManager:
       - arn:aws:iam::aws:policy/SecretsManagerReadWrite
       - arn:aws:iam::aws:policy/AWSCertificateManagerReadOnly
 
-# Role for a user that can only access AWS Athena
+# Role for a user that can only access AWS Athena in the Synapse Dev account
+SsoSynapseDevAthenaUser:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.1/templates/SSO/aws-sso.njk
+  TemplatingContext:
+    customerManagedPolicies:
+      - Name: SynapseAthenaUserAccessPolicy
+  StackName: !Sub '${resourcePrefix}-${appName}-synapse-dev-athena-user'
+  StackDescription: 'Access to AWS Athena in Synapse Dev account'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref SynapseDevAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref synapseDevAthenaGroup
+    permissionSetName: 'athena-user'
```

Comment:
you should make a separate resource called something like `SsoAthenaUser` then reference the ARN from that resource here with something like..
```
!CopyValue [ !Sub '${resourcePrefix}-${appName}-athena-permission-set-arn' ]
```